docs/source/getting-started/first-feature.rst

```diff
@@ -211,6 +211,6 @@ Next Steps
 
 We've gone through the process of setting up a full-stack Daml app and implementing a useful feature end to end.
 As the next step we encourage you to really dig into the fundamentals of Daml and understand its core concepts such as parties, signatories, observers, and controllers.
-You can do that either by :doc:`going through our docs </daml/intro/0_Intro>` or by taking an `online course <https://daml.com/learn/fundamental-concepts>`_.
+You can do that either by :doc:`going through our docs </daml/intro/0_Intro>` or by taking an `online course <https://daml.com/learn/>`_.
```

Comment:
The link should be `https://daml.com/interactive-tutorials/fundamental-concepts`